gateway-server/src/main/java/org/apache/knox/gateway/config/impl/GatewayConfigImpl.java

```diff
@@ -233,6 +233,12 @@
   static final String REMOTE_ALIAS_SERVICE_CONFIG_PREFIX = GATEWAY_CONFIG_FILE_PREFIX + ".remote.alias.service.config.prefix";
   static final String REMOTE_ALIAS_SERVICE_CONFIG_PREFIX_DEFAULT = GATEWAY_CONFIG_FILE_PREFIX + ".remote.alias.service.config.";
 
+  public static final String REFRESHABLE_SERVIVCE_PARAMETERS_FILENAME = "refreshable-service-parameters.xml";
```

Comment:
Since service params are already refreshable via direct descriptor editing, I don't think that needs to be emphasized in the naming herein. This feature is really about updating granular parts of descriptors, and the naming should reflect that.